projects/mercury/README.md

```diff
@@ -1,87 +1,46 @@
-# Mercury
+# Getting Started with Create React App
```

Comment:
You overwrited the original documentation, could you revert this change?

Reply:
yes :)